src/database/user.ts

```diff
@@ -1,12 +1,22 @@
-import db from "./database";
+import User from "../schemas/User";
 
-const collectionName = "users";
+export async function createUser(userData: any) {
```

Comment:
?

Reply:
@kylepham What is your feedback? This function is called in `/routes`.